docs/collecting-data/code-generation/using-the-cli/index.md

```diff
@@ -294,23 +294,23 @@ To add an Event Specification to the code generation, either manually or through
 
 ![event specification id](images/ts-id.png)
 
-Then you should add this ID to your `snowtype.config.json` `eventSpecificationIds` array.
+Then you should add this ID to your `snowtype.config.json`, `snowtype.config.js`, or `snowtype.config.ts` `eventSpecificationIds` array.
```

Comment:
^^ Same with the above, replacing all the possible file extensions with just `configuration file`.